vignettes/using-vimpact.Rmd

```diff
@@ -0,0 +1,134 @@
+---
+title: "Using vimpact for estimating vaccine impact"
+author: "Rob Ashton, Xiang Li, Katy Gaythorpe"
+date: "`r Sys.Date()`"
+output: rmarkdown::html_vignette
+vignette: >
+  %\VignetteIndexEntry{Using vimpact for estimating vaccine impact}
+  %\VignetteEngine{knitr::rmarkdown}
+  %\VignetteEncoding{UTF-8}
+---
+
+```{r setup, include = FALSE}
+knitr::opts_chunk$set(
+  collapse = TRUE,
+  comment = "#>"
+)
+default_reporter <- testthat::default_reporter()
```

Comment:
why are we needing to change the reporter for testthat here (the variable `default_reporter` is also unused)